tests/Jenkins/dirac_ci.sh

```diff
@@ -66,11 +66,11 @@ fi
 
 # Creating default structure
 mkdir -p "$WORKSPACE/TestCode" # Where the test code resides
-TESTCODE=${_}
+readonly TESTCODE=${_}
```

Comment:
Why do we gain with this ?

Reply:
`readonly` just means that this path is a constant. I had an error in the recent past that I traced down to one of these variables being changed, so this would prevent that.

Reply:
Yes but do we actually want that ? I guess being able to change these values is not a bad things !

Reply:
I would not change it, really. You can change the value of `$WORKSPACE` but not the directory structure underneath.